src/useCachedPromise.ts

```diff
@@ -20,6 +30,11 @@ export type CachedPromiseOptions<T extends FunctionReturningPromise, U> = Promis
    * This is particularly useful when used for data for a List to avoid flickering.
    */
   keepPreviousData?: boolean;
+  /**
+   * The hook generates a cache key from the promise & its arguments. Sometimes that's not enough to guarantee,
+   * uniqueness, and in those cases you can pass a `cacheKeySuffix`.
+   */
+  cacheKeySuffix?: string;
```

Comment:
I'm wondering if we have to expose it 🤔

Reply:
we don't _need_ to expose it, no. I'll make it internal-only for now.